### PR TITLE
Initial calendar view setting

### LINKED ISSUE #124: Set custom default view for calendar
Setting default view to daily or other instead of weekly.

### PR DESCRIPTION
**Pull request by MatteoGalletta**
Allows to set the default view in the plugin settings.
This currently works only for desktop, I don't really know how to implement the mobile version since the check gets done relying on the screen dimensions. The first and immediate solution I can think of consists of adding two different rows in the settings (one for mobile, one for desktop), but maybe it could result overwhelming.

Fixes #124

### CODE REVIEW

**Comment by davish** [comment]
Personally, I think that two different rows in Settings makes sense, since folks often want to see different views depending on the device they're on.

**Comment by MatteoGalletta** [comment]
Thanks for the feedback, I added the mobile option too.

File: src/settings.ts [hunk 1]
```diff
@@ -25,13 +25,21 @@ export interface FullCalendarSettings {
...
+	};
 }
 
 export const DEFAULT_SETTINGS: FullCalendarSettings = {
 	calendarSources: [],
 	defaultCalendar: 0,
 	recursiveLocal: false,
 	firstDay: 0,
+	initialView: {
+		desktop: "timeGridDay",
```
Comment (davish) [inline src/settings.ts:40]: The default desktop view should remain `timeGridWeek` to match the behavior before this PR.

**Review by davish — CHANGES_REQUESTED** [review]
Code looks great! One change to make sure the default view doesn't change from Week to Day for everyone on desktop.

File: src/settings.ts (hunk 1 above)
Comment (MatteoGalletta) [reply src/settings.ts:40]: Completely agree.

**Review by davish — CHANGES_REQUESTED** [review]
Sorry for one last request -- I didn't catch this the first time! Thanks again for your work here.

**Review by davish — APPROVED** [review]

**Pull request merged**